src/APIClient/Router.cpp

```diff
@@ -0,0 +1,18 @@
+#include <APIClient/Router.hpp>
+
+std::string APIClient::Router::getUrlString(APIClient::Router::Route route, std::string id = "") {
```

Comment:
A default value for a parameter must be present in the header file, not in the source file.